test/unit/lifecycle/property.js

```diff
@@ -323,5 +323,107 @@ describe('lifecycle/property', function () {
         create(opts);
       });
     });
+
+    describe('change()', function () {
+      it('is not called if the property has no default value', function () {
+        let calls = 0;
+        create({
+          change () {
+            ++calls;
+          }
+        });
+        expect(calls).to.equal(0);
+      });
+
+      it('is called if the property has no default value and it is set', function () {
+        let calls = 0;
+        let elem = create({
```

Comment:
We could use `const` for declarations like this where we don't need to assign again